SKILL.tsv

```diff
@@ -1039,16 +1039,16 @@ SKILL_20150317_001038	Testuto
 SKILL_20150317_001039	Skirmisher
 SKILL_20150317_001040	Change Direction
 SKILL_20150317_001041	Deeds of Valor
-SKILL_20150317_001042	Increase Attack Speed, Attack Rate during hit, Decrease Defense
+SKILL_20150317_001042	Increases your attack speed, increases your attack rate when hit and decreases your defense
```

Comment:
Extremely confusing. 
Judging by skill description, it should be along the lines of "Increases your attack speed and decreases your defense with every attack" (or "with every hit you receive, this part needs clarification).